Configuration/PyReleaseValidation/python/relval_2026.py

```diff
@@ -20,6 +20,7 @@
 numWFIB.extend([23234.21,23434.21]) #2026D49 prodlike, prodlike PU
 numWFIB.extend([23234.103]) #2026D49 aging
 numWFIB.extend([23234.9]) #2026D49 vector hits
+numWFIB.extend([23234.13]) #2026D49 MLPF
```

Comment:
I thought that the current setup was tested so far just for Run2/Run3.
Is it meaningful then to define the default setup for something out of the test domain?

Reply:
fixed in 4d29d267cd6